app/api/views.py

```diff
@@ -13,5 +13,11 @@ def process_coords():
     Returns: This returns a post request of create_segments().
     """
 
-    data = json.loads(request.form.get("coords")).get("b")
-    return "{}".format(create_segments(data))
+    data = json.loads(request.form.get("coords"))
+    count = 1;
```

Comment:
Actually I can see many problems arising from how I do this. Let's say we have 3 regions then delete one. Now we have will still have three hseg keys. I would need to clear the existing cache (session).